### ISSUE TITLE
Stacking in the pits

### ISSUE THREAD

**Issue by JonBeagles** [christopherh0rn/f1-revisited-bugs#81]
This one I'm guessing might need quoting on - one to either have for this season or not FYI (i.e. if its costly and will take time to do, then likely park it for next season):

How easy is it to add to the race that if you pit on the same lap as a team-mate, that you get queued up behind them (i.e. every time that you pit on the same lap as someone else in your team, you have to wait for their stop to finish first)?

e.g. Car 1 pits and takes 30 seconds to complete their pit-stop, Car 2 pits on the same lap and is 5 seconds behind, so must wait 25 seconds before starting their pit-stop (so likely their pit-stop would take 55 seconds instead).

**Comment by christopherh0rn** [comment]
This does not sound simple to implement I'm afraid so would suggest this is parked for now.

**Comment by JonBeagles** [comment]
Yep that's fine - one for next season :)

Get Outlook for Android<https://aka.ms/ghei36>
________________________________
